### ISSUE TITLE
re-implement question

### ISSUE THREAD

**Issue by laihaotao** [michuanhaohao/reid-strong-baseline#18]
Hi,

Thanks to the author's great work, recently I would like to port your implementation into Keras, since I am not familiar with Pytorch.

Unfortunately, I have some issues when I finish the porting. For the standard baseline, the top1 acc I got is poor can only reach round 66%. I am using the ID loss + triplet hard loss. I also went through michuanhaohao's keras-reid repo. 

If anyone can have a quick look and give any suggestions will be highly appreciated.
[https://github.com/laihaotao/reid](https://github.com/laihaotao/reid)

Thanks!

**Comment by laihaotao** [comment]
solved by myself, anyway thanks
